### PR TITLE
Add note about eCDN to view-only-meeting-experience.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/9422.

### CODE REVIEW

**Comment by dariomws** [comment]
@StainlessSteele, do you agree?

**Comment by tiburd** [comment]
reviewed + confirmed

File: Teams/view-only-meeting-experience.md [hunk 1]
```diff
@@ -26,6 +26,9 @@ appliesto:
 > [!Note]
 > If your meeting hits capacity, Teams will seamlessly scale to accommodate a 10,000-person view-only broadcast experience. Plus, during this time of increased remote work, take advantage of even larger 20,000-person broadcasts through the end of this year. Webinars currently don't support a view-only broadcast experience.
 
+> [!Note]
+> A Teams Meeting is not a Teams Live Event (TLE) and will not leverage Microsoft eCDN. For more information, see (Microsoft eCDN onboarding checklist)[/ecdn/integration/onboarding-checklist-for-tle-customers].
```
Comment (JohanFreelancer9) [inline Teams/view-only-meeting-experience.md:30]:
```suggestion
> A Teams Meeting is not a Teams Live Event (TLE) and will not leverage Microsoft eCDN. For more information, see [Microsoft eCDN onboarding checklist](/ecdn/integration/onboarding-checklist-for-tle-customers).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**